docs/porting/port-migrate-and-upgrade-visual-studio-projects.md

```diff
@@ -246,6 +246,7 @@ If you don't see a project or file type listed here that should be, consult the
 | SQL Server Reporting Services and SQL Server Analysis Services (SSRS, SSDT, SSAS, MSAS) | Support for these project types is provided through two extensions in the Visual Studio Gallery:  [Microsoft Analysis Services Modeling Projects](https://marketplace.visualstudio.com/items?itemName=ProBITools.MicrosoftAnalysisServicesModelingProjects) and [Microsoft Reporting Services Projects](https://marketplace.visualstudio.com/items?itemName=ProBITools.MicrosoftReportProjectsforVisualStudio). SSDT support is also included with the Data Storage and Processing workload in Visual Studio 2019. For more information, see the [Download and install SQL Server Data Tools (SSDT) for Visual Studio](/sql/ssdt/download-sql-server-data-tools-ssdt) page. |
 | SQL Server Integration Services (SSIS) | Support for Visual Studio 2019 is available. For more information, see the [Download and install SQL Server Data Tools (SSDT) for Visual Studio](/sql/ssdt/download-sql-server-data-tools-ssdt) page, the [SQL Server Integration Services (SSIS)](https://techcommunity.microsoft.com/t5/SQL-Server-Integration-Services/bg-p/SSIS) team blog, and the [SQL Server Integration Services Projects](https://marketplace.visualstudio.com/items?itemName=SSIS.SqlServerIntegrationServicesProjects&ssr=false#overview) page on the Marketplace. |
 | Test Window Extension | In Visual Studio 2019, some test window APIs that were previously marked public but were never officially documented have been removed. Widely visible APIs were marked deprecated in Visual Studio 2017 to give extension maintainers an early warning. To our knowledge, few extensions have taken a dependency on these APIs. For more info and updates, view the [complete list of deprecated test-related APIs](https://github.com/Microsoft/vstest/issues/1830). If this affects your scenario, please let us know on [developer community](https://aka.ms/feedback/suggest?space=8). |
+| TypeScript | The TypeScript SDK has been deprecated in Visual Studio 2022 and is no longer installed by default in any workload. Projects that compile TypeScript should install the [Microsoft.TypeScript.MSBuild](https://www.nuget.org/packages/Microsoft.TypeScript.MSBuild/) NuGet package. To support projects that cannot be upgraded immediately, the TypeScript SDK is still available as an optional component in the Visual Studio installer, as well as on the [VS Marketplace](https://marketplace.visualstudio.com/items?itemName=TypeScriptTeam.typescript-442). |
```

Comment:
Perhaps we should use the full name Visual Studio Marketplace instead of the abbreviation VS Marketplace?

Reply:
Thanks; I noticed that and was going to fix it after publish; but, if we can fix it beforehand, all to the better.